bigflow/dataflow.py

```diff
@@ -0,0 +1,42 @@
+import typing
+
+from apache_beam import Pipeline
+from apache_beam.options.pipeline_options import PipelineOptions, GoogleCloudOptions
+
+from .workflow import Job, JobContext
+
+
+class BeamJob(Job):
+    def __init__(
+            self,
+            id: str,
+            driver_callable: typing.Callable[[Pipeline, JobContext, typing.Dict], None],
+            pipeline_options: PipelineOptions,
+            driver_arguments: typing.Optional[dict] = None,
+            wait_until_finish: bool = True):
+        self._id = id
+        self.driver_callable = driver_callable
+        self.driver_arguments = driver_arguments
+        self.pipeline_options = pipeline_options
+        self.wait_until_finish = wait_until_finish
+
+    def execute(self, context: JobContext):
+        pipeline_options = self._apply_logging(self.pipeline_options, context.workflow.workflow_id)
+        pipeline = Pipeline(options=pipeline_options)
+        self.driver_callable(pipeline, context, **self.driver_arguments)
+        pipeline = pipeline.run()
+        if self.wait_until_finish:
+            pipeline.wait_until_finish()
+
+    @property
```

Comment:
There is no real need to add this property

Reply:
true